core/src/components/modal/test/sheet/e2e.ts

```diff
@@ -117,3 +117,86 @@ test('input should not be focusable when backdrop is active', async () => {
   const parentEl = await getActiveElement(page);
   expect(parentEl.tagName).toEqual('ION-BUTTON');
 });
+
+describe('modal: sheet: setting the breakpoint', () => {
+
+  let page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage({
+      url: '/src/components/modal/test/sheet?ionic:_testing=true'
+    });
+  });
+
+  describe('setting an invalid value', () => {
+
+    let warnings: string[];
+    let modal: E2EElement;
+
+    beforeEach(async () => {
+      warnings = [];
+      page.on('console', (ev) => {
+        if (ev.type() === 'warning') {
+          warnings.push(ev.text());
+        }
+      });
+
+      modal = await openModal(page, '#sheet-modal');
+
+      await modal.callMethod('setCurrentBreakpoint', 0.01);
+      await page.waitForChanges();
+    });
+
+    it('should not change the breakpoint', async () => {
+      const breakpoint = await modal.callMethod('getCurrentBreakpoint');
+      expect(breakpoint).toBe(0.25);
+    });
+
+    it('should console a warning to developers', async () => {
+      expect(warnings.length).toBe(1);
+      expect(warnings[0]).toBe('[Ionic Warning]: Attempted to set invalid breakpoint value 0.01. Please double check that the breakpoint value is part of your defined breakpoints.');
+    });
+
+  });
+
+  describe('setting the breakpoint to the same value', () => {
+
+    it('should do nothing', async () => {
+      const modal = await openModal(page, '#sheet-modal');
+
+      await modal.callMethod('setCurrentBreakpoint', 0.25);
+      await page.waitForChanges();
+
+      const breakpoint = await modal.callMethod('getCurrentBreakpoint');
```

Comment:
Do we need `waitForChanges()`? I don't know if the test would catch this bug if it regressed. Might be worth intentionally breaking this to test.

Reply:
Confirmed this test isn't covering the behavior. I'm going to end up removing it, as we are validating other invalid inputs and I'll add a test to confirm that the breakpoint value updates when using a correct value.